### DISCUSSION TITLE
Демо пример продажи и склад

### DISCUSSION THREAD

**Discussion by GaleevRuslan** [totumonline/totum-ru-issues-and-discussions#180] [Как это сделать?] [answered]
Добрый день. Подключите плиз демо пример Продажи и склад для доработки под наш бизнес процесс

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Файл примера: https://raw.githubusercontent.com/totumonline/ttm-samples/main/samples/demoru_warning.json.gz.ttm

Инструкции по установке примеров: https://github.com/totumonline/ttm-samples

При установке для роли `Продажи` выбрать `Создать новую`.
Для `Дерева` тоже `Создать новые`. 
Для `Категорий` — `сопоставить` с существующими.

**Comment by GaleevRuslan** [comment]
Алексей, добрый день! Я извиняюсь, может упустил. А мы с вами обсуждали подключение сервера за 100-200 руб.Правда я писал с ***@***.*** system.ruПоможете развернуть сервер? И базу Продажи и склад сразу, если можете загрузите пож-таГотов дополнительно оплатить ваше время. -- Sincerely, Ruslan GaleevVESA System LLC,Russian Federation,Tatarstan RepublicNaberezhnye Chelny-- С уважением,Галеев Руслан Ильгизович,г.Набережные Челнытел. +7 917 253 93 46, +7 995 095 80 12Производство и установка датчиков нагрузки на осьи бортовых систем взвешиванияwww.vesasystem.ruwww.vesa.systems   05.08.2022, 18:02, "Alexey Polunin" ***@***.***>: Файл примера: https://raw.githubusercontent.com/totumonline/ttm-samples/main/samples/demoru_warning.json.gz.ttmИнструкции по установке примеров: https://github.com/totumonline/ttm-samples—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Reply by ttm-alexeypolunin** [reply]
Добро пожаловать в наше облако https://ru.totum.online/servers, мы вам все поставим = 4239 р. /мес

Если вы хотите за 100-200 р. то вы делаете все сами и сами ставите на шаред, например NetAngels: 

1. Инструкция установки: https://ru.docs.totum.online/install_on_net_angels
2. Загружаете Demo: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/180#discussioncomment-3334639

Если облако не вариант и нужно, что бы кто-то на ваш сервер поставил — посмотрите партнеров: https://ru.totum.online/partners

**Comment by GaleevRuslan** [comment]
Без проблем.Просто я не совсем понял вот этот оффер: -- Sincerely, Ruslan GaleevVESA System LLC,Russian Federation,Tatarstan RepublicNaberezhnye Chelny-- С уважением,Галеев Руслан Ильгизович,г.Набережные Челнытел. +7 917 253 93 46, +7 995 095 80 12Производство и установка датчиков нагрузки на осьи бортовых систем взвешиванияwww.vesasystem.ruwww.vesa.systems   05.08.2022, 18:02, "Alexey Polunin" ***@***.***>: Файл примера: https://raw.githubusercontent.com/totumonline/ttm-samples/main/samples/demoru_warning.json.gz.ttmИнструкции по установке примеров: https://github.com/totumonline/ttm-samples—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Reply by ttm-alexeypolunin** [reply]
Я посчитал еще раз наши возможности и стало понятно, что мы как разработчик платформы не можем сами ставить на сторонние сервера — мы можем предложить только облако. Если нужно, что бы кто-то поставил, посмотрите партнеров: https://ru.totum.online/partners или сами по инструкции.

**Comment by GaleevRuslan** [comment]
Не не, мне же облако. Зачем мне вкладываться в сервер, если я не понимаю, получится ли у меня реализовать проект. Мне надо развернуть базу продажи и склад на простейшем облачном сервере чтобы я смог на ней потренироваться, доработать. Если все получится, я перенесу уже на более производительный сервер. -- Sincerely, Ruslan GaleevVESA System LLC,Russian Federation,Tatarstan RepublicNaberezhnye Chelny-- С уважением,Галеев Руслан Ильгизович,г.Набережные Челнытел. +7 917 253 93 46, +7 995 095 80 12Производство и установка датчиков нагрузки на осьи бортовых систем взвешиванияwww.vesasystem.ruwww.vesa.systems10.08.2022, 23:35, "Alexey Polunin" ***@***.***>:
Я посчитал еще раз наши возможности и стало понятно, что мы как разработчик платформы не можем сами ставить на сторонние сервера — мы можем предложить только облако. Если нужно, что бы кто-то поставил, посмотрите партнеров: https://ru.totum.online/partners или сами по инструкции.
